Bottom: 714f0b60cd467be3bf3fcdfb7f21fc600e5e64a9
Top:    74ff189e5bec68ad83c8419b5793be0512e84343
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-21 02:35:44 +0000

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 2b81861..bfda63a 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1239,6 +1239,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1261,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
